PyRDF/backend/Backend.py

```diff
@@ -53,6 +53,12 @@ def __init__(self, config={}):
 
         """
         self.config = config
+        # Users can set a function to initialize the environment
+        self.initialization = None
+
+    def registerInitialization(self, fun, *args, **kwargs):
```

Comment:
I would prefer to have a shorter name for an API method.
What about just `initialize` ?

Reply:
Also, this method needs documentation.